src/components/Breadcrumbs/_Breadcrumbs.scss

```diff
@@ -3,67 +3,83 @@ $break-breadcrumbs-mobile: max-width 599px;
 .breadcrumbs {
     @include breakout;
 
-    background-color: var(--section-research-secondary);
+    background-color: var(--ui-black);
     padding-bottom: var(--space-xs);
     padding-top: var(--space-xs);
-}
 
-.breadcrumbs__list {
-    @include list-reset;
-    @include wrapper;
+    &__item {
+        @include space-inline-xxs;
 
-    color: var(--ui-white);
-}
+        align-items: center;
+        display: inline;
+        font-size: var(--font-size--1);
+        line-height: 1.5;
+        position: relative;
+        word-break: break-word;
 
-.breadcrumbs__item {
-    @include space-inline-xxs;
+        &:not(:last-child) {
+            &::after {
+                content: "/";
+                padding-left: var(--space-xxs);
+            }
 
-    align-items: center;
-    display: inline;
-    font-size: var(--font-size--1);
-    line-height: 1.5;
-    position: relative;
-    word-break: break-word;
+            @include breakpoint($break-breadcrumbs-mobile) {
+                display: none;
+            }
+        }
 
-    &:not(:last-child) {
-        &::after {
-            content: "/";
-            padding-left: var(--space-xxs);
+        &:last-child {
+            @include breakpoint($break-breadcrumbs-mobile) {
+                padding-left: var(--space-m);
+            }
         }
+    }
+
+    &__icon {
+        @include icon;
 
-        @include breakpoint($break-breadcrumbs-mobile) {
+        left: 0;
+        position: absolute;
+        top: 50%;
+        transform: rotate(90deg) translateX(-50%);
+
+        @include breakpoint($breakpoint-medium) {
             display: none;
         }
     }
 
-    &:last-child {
-        @include breakpoint($break-breadcrumbs-mobile) {
-            padding-left: var(--space-m);
-        }
+    &__link,
+    &__link:link,
+    &__link:visited,
+    &__link:focus {
+        color: inherit;
+        text-decoration: none;
     }
-}
 
-.breadcrumbs__icon {
-    @include icon;
+    &__link:hover {
+        text-decoration: underline;
+    }
 
-    left: 0;
-    position: absolute;
-    top: 50%;
-    transform: rotate(90deg) translateX(-50%);
+    &__list {
+        @include list-reset;
+        @include wrapper;
 
-    @include breakpoint($breakpoint-medium) {
-        display: none;
+        color: var(--ui-white);
     }
-}
 
-.breadcrumbs__link,
-.breadcrumbs__link:link,
-.breadcrumbs__link:visited,
-.breadcrumbs__link:focus {
-    color: inherit;
-    text-decoration: none;
-}
+    .nypl--books-and-more & {
```

Comment:
So this is going to be storybook-only?

Reply:
The styles are getting shipped, so no. In order to get these styles into your app you'll need to apply `.nypl--books-and-more` to `<body>` in your `App.js` or something along those lines. I think an update to the README would be appropriate, potentially.